vhdbuilder/scripts/automate_version_bump.sh

```diff
@@ -17,6 +17,19 @@ build_ids=$3
 branch_name=imageBump/$new_image_version
 pr_title="VHDVersion"
 
+find_and_write_build_timestamp() {
+    # This function takes the build ID and reads the queue time.
+    # It then sanitizes the queue time in the format that the Canonical snapshot endpoint expects, which is 20230727T000000Z 
+    # Following that, it will write the timestamp to a JSON file to be consumed later in the event of a hotfix
+    # This file is only written to an official branch, because the build timestamp will correspond with a particular VHD
+
```

Comment:
I'd put these comments above the function to maintain style consistency.

Reply:
done